### DISCUSSION TITLE
start/stop recirculation missing

### DISCUSSION THREAD

**Discussion by rcshropshire** [explosivo22/rinnaicontrolr-ha#61] [General]
I have installed v.1.3.9 and have everything working except the services to start and end the recirculation.  Is there a config variable I need to set?

**Comment by explosivo22** [comment]
I apologize for missing this. Were you able to get this working?

**Comment by dlindahl** [comment]
I am brand new to Home Assistant and I just added this integration. I too am experiencing this. I have 1 device and 12 entities, but no services, just sensors.

<img width="758" alt="Screen Shot 2023-10-27 at 11 38 40 PM" src="https://github.com/explosivo22/rinnaicontrolr-ha/assets/55945/c3d5995f-e1e6-4ad2-8226-b2348d6c5e54">

**Reply by dlindahl** [reply]
Oops, nevermind! I found the services elsewhere in HA. 🙈

**Reply by explosivo22** [reply]
I'm glad you found it.
